cli/command/universeN.go

```diff
@@ -0,0 +1,55 @@
+package command
+
+import (
+	"errors"
+	"log"
+	"strings"
+
+	"github.com/taubyte/dreamland/cli/common"
+	"github.com/taubyte/dreamland/cli/flags"
+	"github.com/urfave/cli/v2"
+)
+
+// Attaches universe argument parsing to a command.
+// `pos` is position of the universe argument, expected to be 0 or 1.
+func UniverseAtPos(c *cli.Command, pos int) {
```

Comment:
Do you need to export it?

Reply:
Oh, thanks. I forgot the meaning of the capital letters. Fixed in 9801c78689234f7bfb6bda89c9d88b011d7f5bab